pom.xml

```diff
@@ -78,7 +78,8 @@
         <properties-maven-plugin.version>1.2.1</properties-maven-plugin.version>
         <selenium.version>3.141.59</selenium.version>
         <spring.security.version>5.7.11</spring.security.version>
-        <flyway-maven-plugin.version>8.4.4</flyway-maven-plugin.version>
+        <!-- https://mvnrepository.com/artifact/org.flywaydb/flyway-maven-plugin -->
```

Comment:
```suggestion
        <!-- Newer versions exist at https://mvnrepository.com/artifact/org.flywaydb/flyway-maven-plugin, but are incompatible with OpenJDK 11. -->
```

Reply:
Is this comment better? Or would you prefer no comment at all?

Reply:
I would use no comment at all.

Reply:
Okay, I removed it now.